src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMenu.js

```diff
@@ -88,7 +88,7 @@ export const BaseCommentMenu = ({
                         <FormattedMessage {...messages.commentResolveMenuItem} />
                     </MenuItem>
                 )}
-                {canEdit && (
+                {canEdit && !isResolved && (
```

Comment:
Is it not possible to have a backend change for this? If canEdit is false if the status is resolved on the backend, then we wouldn't need this extra flag on the FE.

Reply:
Ya I have been looking into EUW endpoints and extending permissions to add `isResolved` to both top level comment and replies. That seems like a better place put these checks. 

That work is currently blocked since EUW local dev setup is broken but we can disregard this PR if that is the direction we agree on.

Reply:
Yeah I agree with the direction of extending permissions. This will also prevent users from bypassing the ui and hitting the endpoint directly.

Reply:
On the back-end canEdit should already be checking resolution status for top level and replies. I would be curious why it isn't coming through.

Reply:
@shahzadaziz Before your changes, did the menu item to modify comment show up when the comment is resolved?